common/src/main/java/uk/gov/gchq/palisade/service/request/DataRequestConfig.java

```diff
@@ -122,7 +122,7 @@ public String toString() {
         return new ToStringBuilder(this)
                 .appendSuper(super.toString())
                 .append("user", user)
-                .append("justification", context)
+                .append("purpose", context)
```

Comment:
This should actually refer to the context. looks like legacy comments that were not changed when we expanded to context rather than just having a justification.